cfgov/unprocessed/apps/ccdb-landing-map/js/TileMap.js

```diff
@@ -21,84 +25,113 @@ export function calculateDateInterval() {
 /* ----------------------------------------------------------------------------
    Utility Functions */
 /**
-* A reducer function to process the maximum value in the state complaint data
-*
-* @param {number} accum the current max value
-* @param {Object} stateComplaint a candidate value
-* @returns {string} the maximum between the current and a state entry
-*/
-export function findMaxComplaints( accum, stateComplaint ) {
-  return Math.max( accum, stateComplaint.displayValue );
+ * Creates N evenly spaced ranges in the data
+ *
+ * @param {Array} data all of the states w/ displayValue, complaintCount, raw
+ * @param {Array} colors an array of colors
+ * @returns {Array} floating point numbers that mark the max of each range
+ */
+export function makeScale( data, colors ) {
+  const allValues = data.map( x => x.displayValue );
+  const uniques = new Set( allValues );
+
+  let scale = d3.scaleQuantile().range( [ WHITE, ...colors ] );
```

Comment:
Would it be possible do all we need while only importing d3-scale, or do we need the entire package?

Reply:
I actually tried that, but was getting some bad errors in the runtime. 
i wanted to do something like:

```
include * as d3scale from 'd3-scale'
include * as d3select from 'd3-selection' 
```

I think to do that you need to create a special bundle with rollup or something